internal/pkg/describe/uri.go

```diff
@@ -409,8 +409,10 @@ func (u *accessURI) strings() []string {
 			protocol = "https://"
 		}
 		path := ""
-		if u.Path != "/" {
```

Comment:
i think this misses a case with a path with multiple slashes, like `///`. that should be solved by just changing the `else if` to an `else` (or drop the `else` like this)!
```suggestion
		path := u.Path
		if !strings.HasPrefix(u.Path, "/") {
		    path = fmt.Sprintf("/%s", u.Path)
		}
```